src/main/conversions/v2.25.0/c2_25_0_2019020601.clj

```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c2-25-0-2019020601
+  (:use [kameleon.sql-reader :only [exec-sql-statement]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.25.0:20190206.01")
```

Comment:
We also have to add this verison to the `src/main/data/99_version.sql` file, for new databases.
```sql
INSERT INTO version (version) VALUES ('2.25.0:20190206.01');
```